Missing margin when using cart and checkout blocks in full width mode

## Describe the bug

* When using the cart block in full width mode, the block touches the left border of the browser.
* When using the checkout block in full width mode, the block touches the left and the right border of the browser.

## To reproduce
Steps to reproduce the behavior:

1. Create a test page and add the cart block.
2. Set the block to full width and save the test page.
3. Look up the cart block in the frontend and see that there's no spacing on the left-hand side.
4. Create another test page and add the checkout block.
5. Set the block to full width and save the test page.
6. Look up the checkout block in the frontend and see that there's no spacing on the left-hand and the right-hand side.

## Expected behavior

I expected that both the cart and the checkout block have some spacing between the block and the browser border.

## Screenshots

<table>
<tr>
<td>Cart:
<br><br>

![cart-without-margin](https://user-images.githubusercontent.com/3323310/139038328-af160c98-d091-47f3-89cd-7444427fbb9f.png)
</td>
<td>Checkout:
<br><br>

![checkout-without-margin](https://user-images.githubusercontent.com/3323310/139038353-44f268bb-6d4f-47cd-b94d-0bd74fb8cc20.png)</td>
</tr>
</table>

@nerrad I created a fresh site using Local by Flywheel and I still do not see the margins on my end. These are the versions I'm running:

* WordPress 5.8.1
* WooCommerce 5.8.0 (via WordPress.org plugin directory)
* WooCommerce Blocks 6.3.0-dev (via local GitHub repo)
* Storefront 3.9.1 (via local GitHub repo)
* Chrome 95.0.4638.69

<details>
  <summary>WooCommerce system status report</summary>

```
### WordPress Environment ###

WordPress address (URL): http://woo.local
Site address (URL): http://woo.local
WC Version: 5.8.0
REST API Version: ✔ 5.8.0
WC Blocks Version: ✔ 6.3.0-dev
Action Scheduler Version: ✔ 3.3.0
WC Admin Version: ✔ 2.7.2
Log Directory Writable: ✔
WP Version: 5.8.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.16.0
PHP Version: 7.3.5
PHP Post Max Size: 1,000 MB
PHP Time Limit: 1200
PHP Max Input Vars: 4000
cURL Version: 7.54.0
LibreSSL/2.6.5

SUHOSIN Installed: –
MySQL Version: 8.0.16
Max Upload Size: 300 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ❌ wp_remote_get() failed. Contact your hosting provider.

### Database ###

WC Database Version: 5.8.0
WC Database Prefix: wp_
Total Database Size: 4.92MB
Database Data Size: 3.47MB
Database Index Size: 1.45MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 2.48MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 0.13MB + Index: 0.06MB + Engine InnoDB
wp_posts: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 24
nav_menu_item: 8
page: 8
post: 1
product: 19
product_variation: 7
revision: 6
shop_order: 1

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ❌Error messages should not be shown to visitors.

### Active Plugins (2) ###

WooCommerce Blocks: by Automattic – 6.3.0-dev
WooCommerce: by Automattic – 5.8.0

### Inactive Plugins (0) ###


### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #78 - /shop/
Cart: #66 - /cart/
Checkout: #68 - /checkout/
My account: #12 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 3.9.1
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 28
Oldest: 2021-11-03 07:55:07 +0000
Newest: 2021-11-03 07:58:39 +0000

Pending: 2
Oldest: 2021-11-04 07:50:03 +0000
Newest: 2021-11-04 07:54:47 +0000


### Status report information ###

Generated at: 2021-11-03 08:04:40 +00:00
```

</details>

I did not install Gutenberg this time and I noticed that you're using Storefront 3.5.1, while I'm using Storefront 3.9.1. When trying to build Storefront 3.5.1, I received the following error in the console:

```
npm run build                                                                                             ✔  12.21.0  

> storefront@3.5.1 build /Users/nielslange/Themes/storefront
> grunt deploy --env=production

Running "stylelint:all" (stylelint) task
>> Deprecation Warning: 'declaration-property-unit-whitelist' has been deprecated. Instead use 'declaration-property-unit-allowed-list'. See: https://github.com/stylelint/stylelint/blob/13.7.0/lib/rules/declaration-property-unit-whitelist/README.md
>> Linted 32 files without errors

Running "sass:dist" (sass) task
Fatal error: Error: It's not clear which file to import for '@import "buttons"'.
       Candidates:
         buttons.scss
         buttons.css
       Please delete or rename all but one of these files.
        on line 1 of assets/css/admin/admin.scss
>> @import "buttons";

   ^

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! storefront@3.5.1 build: `grunt deploy --env=production`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the storefront@3.5.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nielslange/.npm/_logs/2021-11-03T08_16_26_460Z-debug.log
```

>  If we're still not able to determine the cause from comparing environments, the next step might be to get another team member to try and verify, but I can just add this for the next person doing porter rotation to look at.

@frontdevde As you're porter this week, would you mind checking if you're able to reproduce this problem?

--- a/assets/js/blocks/cart-checkout/cart/index.js
+++ b/assets/js/blocks/cart-checkout/cart/index.js
@@ -28,7 +28,7 @@ const settings = {
 	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
 	description: __( 'Shopping cart.', 'woo-gutenberg-products-block' ),
 	supports: {
-		align: [ 'wide', 'full' ],
+		align: [ 'wide' ],
 		html: false,
 		multiple: false,
 		__experimentalExposeControlsToChildren: true,

--- a/assets/js/blocks/cart-checkout/cart/inner-blocks/empty-cart-block/block.json
+++ b/assets/js/blocks/cart-checkout/cart/inner-blocks/empty-cart-block/block.json
@@ -5,7 +5,7 @@
 	"description": "Contains blocks that are displayed when the cart is empty.",
 	"category": "woocommerce",
 	"supports": {
-		"align": [ "wide", "full" ],
+		"align": [ "wide" ],
 		"html": false,
 		"multiple": false,
 		"reusable": false,

--- a/assets/js/blocks/cart-checkout/cart/inner-blocks/filled-cart-block/block.json
+++ b/assets/js/blocks/cart-checkout/cart/inner-blocks/filled-cart-block/block.json
@@ -5,7 +5,7 @@
 	"description": "Contains blocks that are displayed when the cart contains products.",
 	"category": "woocommerce",
 	"supports": {
-		"align": [ "wide", "full" ],
+		"align": [ "wide" ],
 		"html": false,
 		"multiple": false,
 		"reusable": false,

--- a/assets/js/blocks/cart-checkout/checkout/index.tsx
+++ b/assets/js/blocks/cart-checkout/checkout/index.tsx
@@ -27,7 +27,7 @@ const settings = {
 		'woo-gutenberg-products-block'
 	),
 	supports: {
-		align: [ 'wide', 'full' ],
+		align: [ 'wide' ],
 		html: false,
 		multiple: false,
 	},

